Examples/VaporExample/Sources/Run/main.swift

```diff
@@ -1,16 +1,16 @@
 import App
-import MongoSwift
+import MongoDBVapor
 import Vapor
 
 var env = try Environment.detect()
 try LoggingSystem.bootstrap(from: &env)
 
 let app = Application(env)
 try configure(app)
+try app.mongoDB.configure("mongodb://localhost:27017")
```

Comment:
while the parameter isn't required, it seemed useful to me to show it here to make it more clear to a user what they should change if they aren't using the default host/port.